.github/workflows/main.yml

```
@@ -41,6 +41,13 @@ jobs:
           echo 'gem: --no-document' | sudo tee /etc/gemrc
           cd _build
           bundle install
+      - name: Install Pygments
+        run: git clone -b 2.5.2 --depth 1 https://github.com/pygments/pygments.git
+      - name: Install Graphviz lexer for Pygments
+        run: |
+          git clone -b master --depth 1 https://github.com/nikeee/pygments-lexer-graphviz.git
+          cd pygments-lexer-graphviz/
+          sudo python setup.py develop
```

This is a fairly outdated way of installing a Python package from a local clone. I would recommend using

```suggestion
          git clone -b master --depth 1 https://github.com/nikeee/pygments-lexer-graphviz.git
          sudo pip install --editable pygments-lexer-graphviz/
```

If possible I would also leave out the `--editable` flag to `pip` (it shouldn't make a difference in this case).

---

You can also get rid of the `sudo` if you pass `--user` to the `pip` call (not that it makes much of a difference in a container).

---

A one-line version without `sudo` would be
```suggestion
          python3 -m pip install --user git+https://github.com/nikeee/pygments-lexer-graphviz.git@master
```